tests/SideBySide/InsertTests.cs

```diff
@@ -411,6 +411,52 @@ public void InsertBigInteger(bool prepare)
 		Assert.Equal(value, reader.GetValue(0));
 	}
 
+	[Theory]
+	[InlineData(false)]
+	[InlineData(true)]
+	public void InsertMySqlDecimal(bool prepare)
+	{
+		using var connection = new MySqlConnection(AppConfig.ConnectionString);
+		connection.Open();
+		connection.Execute(@"drop table if exists insert_mysql_decimal;
+create table insert_mysql_decimal(rowid integer not null primary key auto_increment, value bigint);");
```

Comment:
`value bigint` seems like the wrong data type.

Reply:
sorry about this stupid mistake